docs/document/content/features/transaction/_index.en.md

```diff
@@ -78,7 +78,7 @@ The following table can be used for comparison to help developers choose the sui
 | Business transformation  | None            | None               | Seata server needed|
 | Consistency    | Not supported         | Not supported             | Final consistency       |
 | Isolation    | Not supported        | Supported           | Business side guaranteed  |
-| Concurrent performance | no loss        | severe loss          | slight loss       |
+| Concurrent performance | no loss        | slight loss          | severe loss       |
```

Comment:
Why change this description?

Reply:
Hi @strongduanmu , is it not required but the information is also not to the point. Let me know if this part of change is not required. Thanks.

Reply:
Thank you very much, but this change looks wrong, so I will close this pr.